css/style.css

```diff
@@ -74,12 +79,167 @@ body {
     padding: 0;
   }
 
-  .connect-me {
+  .connect-two {
     padding-top: 12px;
-    padding-bottom: 15px;
     font-weight: 500;
     font-size: 16px;
     line-height: 24px;
     text-decoration: none;
   }
+
+  #works {
+    display: grid;
+  }
+
+  .work-card {
```

Comment:
As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20in%20%22Works%22%20section%20you%20should%20use%20CSS%20Grid%20and%20Flexbox%20combined%20(it%20is%20mandatory%20that%20you%20use%20both).%20Use%20CSS%20Grid%20in%20the%20main%20container%20that%20contains%20all%20of%20the%20cards%20and%20use%20Flexbox%20to%20lay%20out%20an%20individual%20card.), please implement CSS Flexbox to align the content inside your cards :frog: :+1: